docker/compose/avalon-multi-pool.yaml

```diff
@@ -119,7 +119,7 @@ services:
       - 1948
     command: |
       bash -c "
-        if [ -v CLEAN_SEALED_DATA ] && [ $CLEAN_SEALED_DATA = 1 ]; then rm -f /project/avalon/.sealed/.sealed.data*; fi;
+        if [ ! -v CLEAN_SEALED_DATA ] || [ $CLEAN_SEALED_DATA = 1 ]; then rm -f /project/avalon/.sealed/.sealed.data*; fi;
```

Comment:
Add comment this conditional removing of file.

Reply:
Simplified the logic a bit Ram to make it understandable.